index.html

```diff
@@ -21,156 +21,161 @@
-                  We need lots of bikes and bike accessories! If you have an
-                  old bike you don't use, bring it to one of our dropoff
-                  events. Or come attend one of our fundraisers.
-                </p>
-                <p class="article__read-more">
-                  <a href="#">
-                    <span class="arrow-right"></span>
-                    Learn more
-                  </a>
-                </p>
-              </div>
+        <div>
+          <h2 class="heading-underline">Learn more</h2>
+
+          <div class="articlearea">
+            <div class="article">
+              <h3 class="article__title">Why do refugees need bikes?</h3>
+              <p class="article__summary">
+                Many refugees are placed in housing in areas where there are few
+                jobs. Bikes provide low-cost transportation so that they can get
+                to work, manage child care and do their shopping.
+              </p>
+              <p class="article__read-more">
+                <a href="#">
+                  <span class="arrow-right"></span>
+                  Learn more
+                </a>
+              </p>
             </div>
-          </div>
-        </div>
 
-        <div class="sidebar">
-          <h2 class="heading-underline">Upcoming events</h2>
-
-          <div class="article">
-            <img
-              class="article__thumbnail"
-              src="spring-fundraisers_thumbnail.jpg"
-              alt=""
-            />
-            <div class="article__content">
-              <h3 class="article__title">
-                <a class="article__title-link" href="#">Join us for our Spring fundraisers</a>
-              </h3>
+            <div class="article">
+              <h3 class="article__title">How can I help?</h3>
               <p class="article__summary">
-                Riders needed now to help raise awareness and funds for BfR and
-                Médecins Sans Frontières.
+                We need lots of bikes and bike accessories! If you have an old
+                bike you don't use, bring it to one of our dropoff events. Or
+                come attend one of our fundraisers.
+              </p>
+              <p class="article__read-more">
+                <a href="#">
+                  <span class="arrow-right"></span>
+                  Learn more
+                </a>
               </p>
             </div>
           </div>
+        </div>
+      </div>
 
-          <div class="article">
-            <img
-              class="article__thumbnail"
-              src="bikes-for-refugees_logo.jpg"
-              alt=""
-            />
-            <div class="article__content">
-              <h3 class="article__title">
-                <a class="article__title-link" href="#">Bike dropoff event in Edinburgh on May 11</a>
-              </h3>
-            </div>
+      <div class="sidebar">
+        <h2 class="heading-underline">Upcoming events</h2>
```

Comment:
There is a space before the text, can you remove this? you might need to check on the `padding` you are using at the moment
![Screenshot 2023-09-27 at 10 14 42](https://github.com/CodeYourFuture/bikes-for-refugees/assets/13808341/200ed96b-4e6f-4a4a-9034-d44f2a899d82)
![Screenshot 2023-09-27 at 10 14 49](https://github.com/CodeYourFuture/bikes-for-refugees/assets/13808341/b180a8f9-92e6-4436-826f-9a859b697a53)

Reply:
would you mind checking the project again please? Thank you!